sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -302,6 +302,24 @@ case class CurrentUser() extends LeafExpression with Unevaluable {
   final override val nodePatterns: Seq[TreePattern] = Seq(CURRENT_LIKE)
 }
 
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = """_FUNC_() - connected user name.""",
```

Comment:
Both this comment and the existing comment for `current_user` on L289 above could use some more information, especially now that we have both `current_user` and `session_user` as separate functions. Even if this PR implements both functions as the same implementation, could we at least (1) dig into how the `current_user` function is currently implemented and describe all the cases there, and (2) mention for this `session_user` function that this should always refer to the identity of the invoker?

Reply:
converting to alias since session_user change is not relevent for non-external functions.